CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required (VERSION 3.8.2)
+project(imgui VERSION 1.73.0 LANGUAGES CXX C)
+
+option (IMGUI_EXAMPLES "Build ImGui examples" ON)
```

Comment:
Here and everywhere else the space before brace is not idiomatic. Ideally it would be good to follow the style used in CMake docs and not opinionated style.

Reply:
> space before brace is not idiomatic

There is no "idiom" when it comes to whitespace in CMake files - I personally prefer the space as it makes things easier to read in all of the projects I create, hence why they're here. The whitespace makes no difference, and since there isn't a CMake linter it's hard to enforce things either way.

Reply:
The CMake scripts will be maintained for years to come by many people. I think it's easier to show people CMake docs as an example of how to write scripts than choose an arbitrary, subjective style and allow inconsistencies which will be introduced in future PRs (the spaces are missing already in some lines of the code in your PR, for example).